youtube_dl/extractor/brightcove.py

```diff
@@ -508,7 +510,21 @@ def _extract_urls(webpage):
                 'http://players.brightcove.net/%s/%s_%s/index.html?videoId=%s'
                 % (account_id, player_id, embed, video_id))
 
-        return entries
+        # 	<video data-brightcove-video-id="5320421710001" data-account="245991542" data-player="SJWAiyYWg" data-embed="default" class="video-js" controls itemscope itemtype="http://schema.org/VideoObject">
+        for video in re.findall(r'(?i)(<video[^>]+>)', webpage):
+            attrs = extract_attributes(video)
+                
+            video_id    = dict_get(attrs, ['data-brightcove-video-id', 'data-video-id'])
+            account_id  = attrs.get('data-account')
+            player_id   = attrs.get('data-player')
+            embed       = attrs.get('data-embed')
+
+            if video_id and account_id and player_id and embed:
+                entries.append(
+                    'http://players.brightcove.net/%s/%s_%s/index.html?videoId=%s'
+                    % (account_id, player_id, embed, video_id))
+
+	return entries
```

Comment:
Well, I definitely don't have any other examples. My best guess from the few google hits for "data-brightcove-video-id" is that this is an older specification that is deprecated but some sites still use it.

A separate extractor? That seems like a lot of work and I don't see what the benefit would be? A lot of code duplication, extra maintenance work, and no actual gain? It would just end up doing the same thing?

I'm also not sure if you mean I should have a `BostonGlobeIE(BrightcoveNewIE)` in `bostonglobe.py` and bypass the generic extractor (I guess this could handle the `video_id` problem?), or a `BrightcoveAltVideoTagIE(BrightcoveNewIE)` in `brightcove.py` that handles this special case of brightcove with the `data-brightcove-video-id` instead of `data-video-id`?

All of these choices seem to introduce a lot more complexity than the current proposal (which implements this difference in one line), so I feel like I am misunderstanding you.

Thanks. Sorry for being obtuse.